_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,68 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector as an Integrator. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
+
+## Input for creating an IdentityAttribute
+
+To create an IdentityAttribute as an Integrator for your own Connector, you need to proceed as described in the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case documentation using the following content:
+
+| Property    | Value                                                                                                    |
+| ----------- | -------------------------------------------------------------------------------------------------------- |
+| `@type`     | `"IdentityAttribute"`                                                                                    |
+| `owner`     | `"<Address of your Connector>"`                                                                          |
+| `validFrom` | `"<start of Attribute validity>"`                                                                        |
+| `validTo`   | `"<end of Attribute validity>"`                                                                          |
+| `value`     | Specify an [IdentityAttribute Value]({% link _docs_integrate/attribute-values.md %}#identity-attributes) |
+| `tags`      | `["<additional information 1>", ..., "<additional information m>"]`                                      |
+
+You need to replace the placeholders marked with `<...>` appropriately. In particular, it is necessary that you insert one of the available [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the `value` property. You are not allowed to specify the Address of a Connector other than your own as the value for the `owner` property, as the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case relates to the creation of Attributes for yourself and not for others. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.
+
+In general, you can query your Connector's Address by proceeding as documented in the [Get currently used Identity]({% link _docs_use-cases/use-case-transport-get-currently-used-identity.md %}) use case. If you use the Demo Connector provided by us for testing purposes in the [interactive excerpt of the Connector's API documentation]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}), the Connector's Address is `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD`.
```

Comment:
It's a huge problem that we copy paste the address `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD` everywhere. From time to time I recreate the connector b/c it gets to full with data and then I have to search the whole docs for this.

IMO we could  
1. just remove this and just tell how they find it out
2. insert an api snippet to query the address
3. load this with js